coops-ui/package.json

```diff
@@ -16,6 +16,7 @@
     "core-js": "^3.1.4",
     "http-status-codes": "^1.3.2",
     "lodash.isempty": "^4.4.0",
+    "lodash.isequal": "latest",
```

Comment:
I don't usually see this done because it's more likely to cause breakage between builds if the package changes. I recommend the usual way of specifying a desired major version but allow minor/patch versions.

Reply:
Absolutely, makes sense. I will change that.

Reply:
Resolved?